### DISCUSSION TITLE
How can I adjust the intensity of RandomMotion?

### DISCUSSION THREAD

**Discussion by wqlevi** [TorchIO-project/torchio#1231] [General]
Dear Devs,

As was mentioned in the title, I was trying to simulate motion MRI image which looks like your [demo ](https://torchio.readthedocs.io/transforms/augmentation.html#randommotion). However, either RandomMotion class creates weak signal intensity for motion or RandomGhosting creates too large range of motion (eventhough the intensity is adjustable). 

I'm wondering how can do a motion simulation with motion's signal intensity close that of the original image?

Thannks,

**Comment by fepegar** [comment]
Hi, @wqlevi. The docs should help: https://torchio.readthedocs.io/transforms/augmentation.html#randommotion

**Reply by wqlevi** [reply]
Hi @fepegar , thanks for your reply but unfortunately this doc does not describe how to adjust the intensity of the simulated motion (by intensity I mean voxel/pixel intensity values), and neither does the source code. Thus, I'm seeking a chance here if you know better about the underlying parameters to change.

**Reply by fepegar** [reply]
Sorry I wasn't clear. Maybe I'm misunderstanding. You can play with these parameters to control the intensity.

![image](https://github.com/user-attachments/assets/66cac321-5b48-402e-bcf3-154cc252c45b)

If this is not enough, maybe you can create a custom transform that outputs a weighted sum of the input and the output.
